tests/cxx/iwyu_stricter_than_cpp.cc

```diff
@@ -113,7 +125,17 @@ void TestTypeAliases() {
   TplDoesEverythingRightAgainAl* tdora_ptr = 0;
   // ...at least until we dereference the pointer
   // IWYU: IndirectStruct2 is...*iwyu_stricter_than_cpp-i2.h
-  (void) dor_ptr->a;
+  (void)dor_ptr->a;
+
+  // Nested name testing
```

Comment:
Very nice. I had several drafts for name change suggestions, but this segment really drives home the full matrix of outer/nested type providing or not. This looks right.